managed/models/artifact_model.go

```diff
@@ -51,13 +52,15 @@ type BackupStatus string
 
 // BackupStatus status (in the same order as in artifacts.proto).
 const (
-	PendingBackupStatus        BackupStatus = "pending"
-	InProgressBackupStatus     BackupStatus = "in_progress"
-	PausedBackupStatus         BackupStatus = "paused"
-	SuccessBackupStatus        BackupStatus = "success"
-	ErrorBackupStatus          BackupStatus = "error"
-	DeletingBackupStatus       BackupStatus = "deleting"
-	FailedToDeleteBackupStatus BackupStatus = "failed_to_delete"
+	PendingBackupStatus             BackupStatus = "pending"
+	InProgressBackupStatus          BackupStatus = "in_progress"
+	PausedBackupStatus              BackupStatus = "paused"
+	SuccessBackupStatus             BackupStatus = "success"
+	ErrorBackupStatus               BackupStatus = "error"
+	DeletingBackupStatus            BackupStatus = "deleting"
+	FailedToDeleteBackupStatus      BackupStatus = "failed_to_delete"
+	RetentionInProgressBackupStatus BackupStatus = "retention_in_progress"
+	FailedRetentionBackupStatus     BackupStatus = "failed_retention"
```

Comment:
Same as above, I recommend 
```suggestion
	CleanupInProgressStatus BackupStatus = "cleanup_in_progress"
	CleanupFailedStatus     BackupStatus = "cleanup_failed"
```